<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddAccount</class>
 <widget class="QDialog" name="DialogAddAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="grpBoxAccountType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>495</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Tipo di Conto</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>33</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioBancario">
       <property name="text">
        <string>Conto Bancario</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btnGrpAccountType</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioCorrente">
       <property name="text">
        <string>Conto Corrente</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btnGrpAccountType</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioRisparmio">
       <property name="text">
        <string>Conto Risparmio</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btnGrpAccountType</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpBoxSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>495</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Impostazioni</string>
   </property>
   <widget class="QCheckBox" name="chkDontAdmitDuplicates">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Non aggiungere se duplicato</string>
    </property>
   </widget>
   <zorder>grpBoxAccountType</zorder>
   <zorder>grpBoxAccountType</zorder>
   <zorder>grpBoxAccountType</zorder>
   <zorder>grpBoxAccountData</zorder>
   <zorder>grpBoxAccountType</zorder>
   <zorder>grpBoxAccountType</zorder>
   <zorder>grpBoxAccountType</zorder>
   <zorder>chkDontAdmitDuplicates</zorder>
  </widget>
  <widget class="QGroupBox" name="grpBoxAccountData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>73</y>
     <width>495</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Dati Conto</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>471</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="lblBalance">
       <property name="text">
        <string>Saldo iniziale:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtBalance">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtSurname"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Nome intestatario:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblSurname">
       <property name="text">
        <string>Cognome intestatario:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAddAccount</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAddAccount</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="btnGrpAccountType"/>
 </buttongroups>
</ui>
